KNet/KNet.API/Models/User.cs

```diff
@@ -1,12 +1,20 @@
-﻿namespace KNet.API.Models
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace KNet.API.Models
 
 {
     public class User : Entity
     {
+        [Required]
         public string FirstName { get; set; }
+        [Required]
         public string LastName { get; set; }
+        [Required]
+        [EmailAddress]
         public string Email { get; set; }
+        [Required]
```

Comment:
Can we check if valid phone number? Maybe we can do it somewhere else (frontend perhaps?)